lib/vtls/sectransp.c

```diff
@@ -2273,6 +2265,38 @@ static CURLcode verify_cert(const char *cafile, struct Curl_easy *data,
   }
 }
 
+static CURLcode verify_cert(const char *cafile,
+                            const struct curl_blob *ca_info_blob,
+                            struct Curl_easy *data, SSLContextRef ctx)
```

Comment:
I think `struct Curl_easy *data` should be the first argument, as that's what most functions do that have such an input.

Reply:
data is now first arg of both verify_cert and verify_cert_buf. Not that verify_cert of current curl release don't have data as first arg.